h/auth/__init__.py

```diff
@@ -21,6 +23,7 @@
 PROXY_POLICY = RemoteUserAuthenticationPolicy(environ_key='HTTP_X_FORWARDED_USER',
                                               callback=groupfinder)
 SESSION_POLICY = SessionAuthenticationPolicy(callback=groupfinder)
+TICKET_POLICY = pyramid_authsanity.AuthServicePolicy()
```

Comment:
As mentioned earlier, I think it would be good to add a paragraph here to explain what all these millions of authentication policies do and how the migration is going to happen.